scripts/devtech/devtech_machines.zs

```diff
@@ -29,10 +32,570 @@ var stock_manipulator = IRecipeMap.recipeMapBuilder("stock_manipulator")
+.outputs(<gregtech:ore_tantalite_0> * 32)
+.buildAndRegister();
+
+stock_manipulator.recipeBuilder()
+.notConsumable(intCircuit(44))
+.outputs(<gregtech:ore_wulfenite_0> * 32)
+.outputs(<gregtech:ore_molybdenite_0> * 32)
+.outputs(<gregtech:ore_powellite_0> * 32)
+.buildAndRegister();
+
+stock_manipulator.recipeBuilder()
+.notConsumable(intCircuit(45))
+.inputs(<contenttweaker:coin_stainless_steel> * 20)
+.EUt(270)
+.duration(200)
+.outputs(<gregtech:ore_silver_0> * 32)
+.outputs(<gregtech:ore_precious_metal_0> * 32)
+.outputs(<gregtech:ore_gold_0> * 32)
+.buildAndRegister();
+
+stock_manipulator.recipeBuilder()
+.notConsumable(intCircuit(46))
+.inputs(<contenttweaker:coin_stainless_steel> * 10)
+.EUt(810)
+.duration(100)
+.outputs(<gregtech:ore_bauxite_0> * 32)
+.outputs(<gregtech:ore_aluminium_0> * 32)
+.outputs(<gregtech:ore_ilmenite_0> * 32)
+.outputs(<gregtech:ore_rutile_0> * 32)
+.buildAndRegister();
+
+stock_manipulator.recipeBuilder()
+.notConsumable(intCircuit(47))
+.outputs(<gregtech:poor_ore_scheelite_0> * 32)
+.outputs(<gregtech:poor_ore_tungstate_0> * 32)
+.outputs(<gregtech:ore_lithium_0> * 32)
+.outputs(<gregtech:ore_tellurium_0> * 32)
+.buildAndRegister();
+
+stock_manipulator.recipeBuilder()
+.notConsumable(intCircuit(48))
+.inputs(<contenttweaker:coin_stainless_steel> * 10)
+.EUt(270)
+.duration(100)
+.outputs(<gregtech:ore_pitchblende_0> * 32)
+.buildAndRegister();
+
+stock_manipulator.recipeBuilder()
+.notConsumable(intCircuit(49))
+.inputs(<contenttweaker:coin_stainless_steel> * 32)
+.EUt(810)
+.duration(320)
+.outputs(<gregtech:ore_scheelite_0> * 32)
+.outputs(<gregtech:ore_tungstate_0> * 32)
+.outputs(<gregtech:ore_lithium_0> * 32)
+.outputs(<gregtech:ore_tellurium_0> * 32)
+.buildAndRegister();
+
+stock_manipulator.recipeBuilder()
+.notConsumable(intCircuit(50))
+.inputs(<contenttweaker:coin_stainless_steel> * 32)
+.EUt(810)
+.duration(320)
+.outputs(<gregtech:ore_scheelite_0> * 32)
+.outputs(<gregtech:ore_tungstate_0> * 32)
+.outputs(<gregtech:ore_lithium_0> * 32)
+.outputs(<gregtech:ore_tellurium_0> * 32)
+.buildAndRegister();
+
+recipes.addShaped("gt_coin_iron", <metaitem:dustIron> * 32, [
+    [<contenttweaker:coin_bronze>, <contenttweaker:coin_bronze>, <contenttweaker:coin_bronze>],
+    [null, null, null],
+    [null, null, null]
+]);
+recipes.addShaped("gt_coin_copper", <metaitem:dustCopper> * 32, [
+    [null, null, null],
+    [<contenttweaker:coin_bronze>, <contenttweaker:coin_bronze>, <contenttweaker:coin_bronze>],
+    [null, null, null]
+]);
+recipes.addShaped("gt_coin_tin", <metaitem:dustTin> * 32, [
+    [null, null, null],
+    [null, null, null],
+    [<contenttweaker:coin_bronze>, <contenttweaker:coin_bronze>, <contenttweaker:coin_bronze>]
+]);
+recipes.addShaped("gt_coin_redstone", <minecraft:redstone> * 32, [
+    [null, null, null],
+    [null, <contenttweaker:coin_bronze>, null],
+    [null, null, null]
+]);
+recipes.addShaped("gt_coin_silver", <metaitem:dustSilver> * 32, [
```

Comment:
IIRC silver was maybe going to be gated later than steam?

Reply:
I think we decided against that, ask @Syrcan